bindings/java/src/lib.rs

```diff
@@ -38,10 +38,21 @@ pub extern "system" fn Java_org_apache_opendal_Operator_getOperator(
         .get_string(&input)
         .expect("Couldn't get java string!")
         .into();
+
     let schema = Scheme::from_str(&input).unwrap();
```

Comment:
```suggestion
     let scheme = Scheme::from_str(&input).unwrap();
```

Reply:
Hello, please refrain from raising reviews related to formatting. These errors should be caught by our CI system. If the CI system fails to catch them, it indicates a bug in our infrastructure.